### ISSUE TITLE
Using a built in function as an argument name and then calling that same function inside will cause an error on load

### ISSUE THREAD

**Issue by KormexGit** [YoYoGames/GameMaker-Bugs#2296]
### Description
While it makes sense that doing this could cause an error, the error message given is vague and does not show the location the error is coming from. Honestly I'm surprised GM allows using built in functions as argument names to start with, but if you don't try to call the function then it works as if it's an argument.

### Steps To Reproduce
This example uses show_debug_message, but it will happen for any built in function.
```
function func(show_debug_message) {
	show_debug_message("Hi");
}
```
This error message appears mid-compile
```
ERROR in
action number 1
of Create Event
for object <undefined>:


Error on load
Unable to find function argument0
```

[fc34c885-b566-4435-804c-23c3a63ae30f](https://api.gamemaker.io/api/github/downloads/fc34c885-b566-4435-804c-23c3a63ae30f)

**Comment by rwkay** [comment]
Fixed in 2024.2

**Comment by scott-dunbar** [comment]
Verified fixed in  IDE v2024.200.0.451  Runtime v2024.200.0.461
